containers/Comments/Comment.js

```diff
@@ -102,10 +97,12 @@ export const Comment = (props) => {
 
   const handleRate = async(ratingComment, value) => {
     if (processing) return;
```

Comment:
still not possible to rate comments. Think `if (processing) return;` should be `if (processing.current) return;`?
empty objects are truthy, so in this case it will always early return.

Reply:
arg, nice catch! fixed